components/cloak-visual.coffee

```diff
@@ -98,7 +98,7 @@ export default
 		# is of a format that woulds support transparency.  The latter is handy
 		# for product images.
 		placeholderColor = unless props.noPlaceholder or
-			(props.autoNoPlaceholder and props.image.match /\.(png|svg)/i)
+			(props.autoNoPlaceholder and props.image?.match /\.(png|svg)/i)
```

Comment:
I guess you could hit this condition with an empty `image` if only a `video` was was provided ...